requirements.txt

```diff
@@ -1,17 +1,15 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Please do not change requirements file.

Testing requirements will be updated in #48